pytest_html/plugin.py

```diff
@@ -278,41 +278,43 @@ def append_extra_html(self, extra, extra_index, test_index):
                 )
                 self.links_html.append(" ")
 
-        def append_log_html(self, report, additional_html):
+        def append_log_html(self, report, additional_html, pytest_capture_value):
             log = html.div(class_="log")
-            if report.longrepr:
-                # longreprtext is only filled out on failure by pytest
-                #    otherwise will be None.
-                #  Use full_text if longreprtext is None-ish
-                #   we added full_text elsewhere in this file.
-                text = report.longreprtext or report.full_text
-                for line in text.splitlines():
-                    separator = line.startswith("_ " * 10)
-                    if separator:
-                        log.append(line[:80])
-                    else:
-                        exception = line.startswith("E   ")
-                        if exception:
-                            log.append(html.span(raw(escape(line)), class_="error"))
+
+            if pytest_capture_value != "no":
```

Comment:
Since this `if` has no else or its own "body", you can probably do:

```python
if pytest_capture_value != "no" and report.longrepr:
```

Reply:
I don't think we could do that, since this `if` also contains the `for section in report.sections:` block. We'd have to duplicate the statement then which is confusing IMO